File: CodeTest/Python/BaekJoon/Class1_14_2920.py
import sys

num_list = list(map(int, sys.stdin.readline().split()))
count = 0

for i in range(1,len(num_list)):
    if num_list[i] == num_list[i-1]+1:
        count += 1
    elif num_list[i] == num_list[i-1]-1:
        count -= 1

if count == 7:
    print('ascending')
elif count == -7:
    print('descending')
else:
    print('mixed')

File: CodeTest/Java/BaekJoon/Class1_14_2920.java
package java_testPackage;

import java.util.Scanner;

public class Class1_14_2920 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int pre = sc.nextInt();
		String sign = "mixed";
		
		for (int i=1; i<8; i++) {
			int num = sc.nextInt();
			
			if(pre+1 == num || pre-1 == num) {
				pre = num;
				
				if(i == 7 && num == 8) {
					sign = "ascending";
				} else if(i==7 && num == 1) {
					sign = "descending";
				}
			} else {
				break;
			}
		}
		
		System.out.println(sign);		
	}

}
